samples/python-v2-azurefunction/README.md

```diff
@@ -62,19 +62,38 @@ The Python v2 programming model introduces the concept of blueprints. A blueprin
 
 # Step 2 - Run Function App with Dapr
 
-Build the function app:
+#### Build the function app:
 
 ```
 dotnet build -o bin/ extensions.csproj
 ```
 
-Note that this extensions.csproj file is required in order to reference the exception as a project rather than as an nuget package. To do the equivalent step with a published version of the extension on nuget.org, run the following step:
+Alternatively, you can install the Dapr extension by running the following command. Retrieve the latest `version` from [here](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Dapr).
 
 ```
 func extensions install -p Microsoft.Azure.WebJobs.Extensions.Dapr -v <version>
 ```
 
-Run function host with Dapr. `--resources-path` flag specifies the directory stored all Dapr Components for this sample. They should be language ignostic.
+#### Create and activate a virtual environment
+
+```PowerShell
+py -m venv .venv
```

Comment:
Is `python` aliased to `py`? Should we just use `python`?

Reply:
Yes, this is same as [MS documentation](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-cli-python?tabs=macos%2Cpowershell%2Cazure-cli&pivots=python-mode-decorators#create-venv) and have been validated.

Reply:
Since it is a Windows specific command, I would suggest to use `python`, which is a valid command for both Linux and Windows.

![image](https://github.com/Azure/azure-functions-dapr-extension/assets/22132944/e08513aa-ecb6-43f2-9bd3-055cc4ab53d5)